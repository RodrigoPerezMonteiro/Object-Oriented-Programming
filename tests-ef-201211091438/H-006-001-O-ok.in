testH4.dat
2
a@a.edu
linguado grelhado
1
0
testH6b.dat
