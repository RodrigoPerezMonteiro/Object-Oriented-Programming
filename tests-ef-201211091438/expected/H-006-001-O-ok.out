Ficheiro a abrir: Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Identificador do cliente (e-mail): Prato do dia a encomendar (nome): Número de doses a encomendar: Adicionar ao carrinho de compras: Operação inválida: O prato pretendido com o nome 'linguado grelhado' não é apropriado para o client 'a@a.edu'.
Gestão de Vendas
1 - Visualizar pratos do dia
2 - Adicionar ao carrinho de compras
3 - Pagar
0 - Sair
Escolha uma opção: Ficheiro a guardar: 